soh/soh/Enhancements/randomizer/3drando/location_access/locacc_ice_cavern.cpp

```diff
@@ -88,9 +99,19 @@ void RegionTable_Init_IceCavern() {
                   LOCATION(RC_ICE_CAVERN_MQ_GS_SCARECROW,     logic->CanUse(RG_SCARECROW) || (logic->IsAdult && (logic->CanUse(RG_LONGSHOT) || ctx->GetTrickOption(RT_ICE_MQ_SCARECROW)))),
   }, {
                   //Exits
-                  Entrance(RR_ICE_CAVERN_MQ_HUB,          {[]{return logic->BlueFire();}}),
+                  Entrance(RR_ICE_CAVERN_MQ_HUB,           {[]{return logic->BlueFire();}}),
                   //Assumes RR_ICE_CAVERN_MQ_HUB access for a pot if using blue fire
-                  Entrance(RR_ICE_CAVERN_MQ_STALFOS_ROOM, {[]{return logic->IsAdult && logic->BlueFire();}}),
+                  Entrance(RR_ICE_CAVERN_MQ_WEST_CORRIDOR, {[]{return logic->IsAdult && logic->BlueFire();}}),
+  });
+
+  areaTable[RR_ICE_CAVERN_MQ_WEST_CORRIDOR] = Region("Ice Cavern MQ West Corridor", "Ice Cavern", {RA_ICE_CAVERN}, NO_DAY_NIGHT_CYCLE, {}, {
+                  //Locations
+                  LOCATION(RC_ICE_CAVERN_MQ_PUSH_BLOCK_POT_1, logic->CanBreakPots()),
+                  LOCATION(RC_ICE_CAVERN_MQ_PUSH_BLOCK_POT_2, logic->CanBreakPots()),
+  }, {
```

Comment:
I'm not seeing any entrances leading to this area.

Reply:
look literally 3 lines up XD